src/PanResponderAdapter.tsx

```diff
@@ -81,22 +82,27 @@ export default function PanResponderAdapter<T extends Route>({
 
       const { timing, ...transitionConfig } = DefaultTransitionSpec;
 
-      Animated.parallel([
-        timing(panX, {
-          ...transitionConfig,
-          toValue: offset,
-          useNativeDriver: false,
-        }),
-      ]).start(({ finished }) => {
-        if (finished) {
-          onIndexChangeRef.current(index);
-          pendingIndexRef.current = undefined;
-        }
-      });
+      if (animationEnabled) {
```

Comment:
This breaks the swipe gesture. The animation should only be disabled when not swiping similar to how it works on Android & iOS

Reply:
Yes, you're right, I didn't noticed that, fixed ✅